README.md

```diff
@@ -18,6 +18,11 @@ This tool supports compiling a workspace for all combinations of the following:
   * ROS 2: `dashing`, `eloquent`
 * OS: `Ubuntu`, `Debian`
 
+NOTE: ROS 2 supports Debian only as a Tier 3 platform.
+This means that there are not `apt` repositories available for the ROS 2 Core on this platform.
```

Comment:
...no `apt`...

Reply:
That would also have been fine, but mine is grammatically correct too